commit 04cf7dc199e379b8248a798467a8f44d1f25743b (refs/changes/99/2599/3)
Author: Nick Zivkovic <nick.zivkovic@joyent.com>
Date:   2017-09-18T20:49:50+00:00 (2 years, 1 month ago)
    
    DOCKER-1107 Docker needlessly fails an assertion
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/lib/backends/sdc/networks.js b/lib/backends/sdc/networks.js
index 38fb59e..04eac3a 100644
--- a/lib/backends/sdc/networks.js
+++ b/lib/backends/sdc/networks.js
@@ -384,7 +384,7 @@ function getNetworksOrPools(params, opts, callback) {
             cb(null, {res: []});
         }
     ], function (err, res) {
-        assert(res.err || res.data);
+        assert(res.err || res.res);
         callback(res.err, res.res);
     });
 }
